index.js

```diff
@@ -1,11 +1,15 @@
-const alertSession = require('./lib/alertSession')
 const ALERT_STATE = require('./lib/alertStateEnum')
-const braveAlerter = require('./lib/braveAlerter')
```

Comment:
Because `alertSession` and `braveAlerter` are classes, changed this so that they start with a capital letter.